CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
   parameter, which takes a `Widget` instead of `IconData`.
   Before: `DescribedFeatureOverlay(icon: Icons.add, ..)`
   After: `DescribedFeatureOverlay(tapTarget: const Icon(Icons.add), ..)`
+* **Breaking change**: `title` and `description` parameters now accept a `Widget` object
```

Comment:
```suggestion
* **Breaking change**: `title` and `description` parameters now take a `Widget`.
```

Reply:
Your English is probably better than mine so I trust you on this one

Reply:
Not too sure about that. The main thing that bugged me was the `object` part, not because it is "wrong" or anything but because it sounds kinda weird to say. In the Flutter world we just refer to `Widget`s as `Widget`s 😆 It is just personal choice. I wanted to emphasize that `String`s are **not** valid anymore. "accept" sounds a bit like it would still accept `String`s.

Reply:
> The main thing that bugged me was the object part, not because it is "wrong" or anything but because it sounds kinda weird to say. In the Flutter world we just refer to Widgets as Widgets 😆 

You have to admit though, that at the end of the day, Widget does extend the Object class ;)

Reply:
"Object" also means instance of a class, I think, so it would make sense either way, it is just w/e 😆